MODEL-METADATA.md

```diff
@@ -38,7 +38,18 @@ which label will be chosen as the predicted label.
 ## Options specific to tasks
 * trainOnProject (optional): A hash with the pid of a project you would like to train your new model or version 
 on. If this is supplied, the code you supplied will start to run against this pid automagically. 
+* userCredentialSpecifications (optional): This is a list of credentials that will be injected from 
```

Comment:
You will also want to make a docs ticket to have this added to the custom task documentation.